### PR TITLE
Add windows service delay

### PR DESCRIPTION
**Pull request by Alberic-Hardis**
documents https://github.com/nats-io/nats-server/pull/3743

### CODE REVIEW

File: running-a-nats-service/running/windows_srv.md [hunk 1]
```diff
@@ -28,3 +28,13 @@ nats-server.exe --signal stop=<service name>
 
 For a complete list of signals, see [process signaling](../nats_admin/signals.md).
 
+## Windows service specific settings
+
+### `NATS_STARTUP_DELAY` environment variable
+
+The Windows service system requires communication with programs that run as Windows services. One important signal form the program is the initial "ready" signal, where the program informs Windows that it is running as expected.
```
Comment (Jarema) [inline running-a-nats-service/running/windows_srv.md:35]:
```suggestion
The Windows service system requires communication with programs that run as Windows services. One important signal from the program is the initial "ready" signal, where the program informs Windows that it is running as expected.
```

Comment (Jarema) [inline running-a-nats-service/running/windows_srv.md:35]:
```suggestion
The Windows service system requires communication with programs that run as Windows services. One important signal from the program is the initial "ready" signal, where the program informs Windows that it is running as expected.
```

**Review by Jarema — CHANGES_REQUESTED** [review]
Thanks for updating the docs.

Found a typo.

**Review by gcolliso — APPROVED** [review]
lgtm - thanks @Alberic-Hardis

**Pull request merged**